portfolio/src/main/java/com/google/sps/data/CommentsData.java

```diff
@@ -0,0 +1,37 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+import static com.googlecode.objectify.ObjectifyService.ofy;
+
+import java.util.List;
+import com.google.sps.data.Comment;
+
+/** Class that allows to perform operations to the comments in the database. */
+public class CommentsData {
```

Comment:
nit: A better name might be `CommentsStore` or `CommentsDataStore` to make it more obvious that this is a storage interface and not just a collection of data.

Reply:
Ok, in fact I was very uncertain about the name of the class.